code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -647,43 +647,37 @@
 	name = "Syndicate Ship Crew Member"
 	roundstart = FALSE
 	death = FALSE
+	show_flavour = FALSE
 	icon = 'icons/obj/machines/sleeper.dmi'
 	icon_state = "sleeper_s"
 	short_desc = "You are a syndicate operative on old ship, stuck in hostile space."
 	flavour_text = "Your ship docks after a long time somewhere in hostile space, reporting a malfunction. You are stuck here, with Nanotrasen station nearby. Fix the ship, find a way to power it and follow your captain's orders."
 	important_info = "Obey orders given by your captain. DO NOT let the ship fall into enemy hands."
 	outfit = /datum/outfit/syndicatespace/syndicrew
-	assignedrole = "Cybersun Crewmember"
-
-/obj/effect/mob_spawn/human/syndicatespace/Initialize(mapload)
-	. = ..()
-	var/policy = get_policy(ROLE_SYNDICATE_CYBERSUN)
-	if(policy)
-		important_info = policy
+	assignedrole = "Cybersun Space Syndicate"
```

Comment:
you should set this string as the define for `ROLE_SYNDICATE_CYBERSUN` (and similarly for the captain), then you can have in special `get_policy(assignedrole)` so you aren't copypasting for the captain

Reply:
"Define" will change it in game to "Cybersun Space Syndicate" anyway, right?

Reply:
It's done.